extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -110,12 +110,17 @@ export class PinWidget {
       'data-pin-log': 'embed_pin',
     }});
 
+    // If no alternate text is set, set it to the title gotten from the pin data
+    if (!this.alt)
```

Comment:
We follow [Google's JS Style Guide](https://google.github.io/styleguide/jsguide.html) which requires braces for all cases.

Reply:
Done.